docs/source/example-c.rst

```diff
@@ -9,79 +9,155 @@ To simplify package management in the example,
    *heat* model from source
 #. Create a *babelizer* input file describing the *heat* model
-#. Run the *babelizer* to generate Python bindings, then build the bindings
+#. Run the *babelizer* to generate a Python package, then build and install the package
 #. Show the *heat* model running in Python through *pymt*
 
 Before we begin,
 create a directory to hold our work:
 
 .. code:: bash
 
-  $ mkdir build && cd build
+  mkdir example-c && cd example-c
 
 This directory is a starting point;
-we'll make new directories under it as we proceed through the example.
-In the end,
-the directory structure under ``build`` should look like that in Figure 1.
+we'll add files and directories to it as we proceed through the example.
+At the end,
+the directory structure under ``example-c`` should look similar to that below.
 
-.. figure:: _static/babelizer-bmi-example-c.png
-    :align: center
-    :alt: Directory structure after completing example
+.. code:: bash
+
+  example-c
+  ├── babel_heatc.toml
+  ├── bmi-example-c
+  │   ├── CMakeLists.txt
+  │   ├── LICENSE
+  │   ├── README.md
+  │   ├── heat
+  │   │   ├── CMakeLists.txt
+  │   │   ├── bmi_heat.c
+  │   │   ├── bmi_heat.h
+  │   │   ├── bmi_main.c
+  │   │   ├── bmiheatc.pc.cmake
+  │   │   ├── heat.c
+  │   │   ├── heat.h
+  │   │   ├── heatc.pc.cmake
+  │   │   └── main.c
+  │   └── testing
+  │       ├── CMakeLists.txt
+  │       ├── config.txt
+  │       ├── test_conflicting_instances.c
+  │       ├── test_get_value.c
+  │       ├── test_grid_info.c
+  │       ├── test_initialize_from_file.c
+  │       ├── test_irf.c
+  │       ├── test_print_var_names.c
+  │       ├── test_reinitialize.c
+  │       └── test_set_value.c
+  ├── environment-c.yml
+  ├── pymt_heatc
+  │   ├── CHANGES.rst
+  │   ├── CREDITS.rst
+  │   ├── LICENSE.rst
+  │   ├── MANIFEST.in
+  │   ├── README.rst
+  │   ├── babel.toml
+  │   ├── build
+  │   │   └── HeatC
+  │   │       └── api.yaml
+  │   ├── docs
+  │   │   ├── Makefile
+  │   │   ├── _static
+  │   │   │   ├── logo-dark.svg
+  │   │   │   └── logo-light.svg
+  │   │   ├── api
+  │   │   ├── authors.rst
+  │   │   ├── babel.rst
+  │   │   ├── changelog.rst
+  │   │   ├── conf.py
+  │   │   ├── developer_install.rst
+  │   │   ├── environments.rst
+  │   │   ├── index.rst
+  │   │   ├── license.rst
+  │   │   ├── quickstart.rst
+  │   │   ├── updating.rst
+  │   │   └── usage.rst
+  │   ├── meson.build
+  │   ├── meta
+  │   │   └── HeatC
+  │   │       └── api.yaml
+  │   ├── noxfile.py
+  │   ├── pymt_heatc
+  │   │   ├── __init__.py
+  │   │   ├── _bmi.py
+  │   │   ├── _version.py
+  │   │   ├── data -> ../meta
+  │   │   └── lib
+  │   │       ├── __init__.py
+  │   │       └── heatc.pyx
+  │   ├── pyproject.toml
+  │   ├── requirements-build.txt
+  │   ├── requirements-library.txt
+  │   ├── requirements-testing.txt
+  │   ├── requirements.txt
+  │   └── setup.cfg
+  └── test
+      └── config.txt
+
+  16 directories, 60 files
```

Comment:
That's a lot of files. If we're going to keep this, maybe reduce the number of files printed to match the image we had for the JOSS review?

So the following for the top-level folder,
```bash
tree example-c -L 1 -F -a
```
and then the following for a more detailed view of the *bmi-example*?
```bash
tree example-c/bmi-example-c -L 1 -F -a
```

Or whatever's most useful to a reader.

Reply:
Check out f571e71